Qualification/Input/Content/References.md

```diff
@@ -2,11 +2,11 @@
 
 **Cho 2002** Cho JY, Yu KS, Jang IJ, Yang BH, Shin SG, Yim DS. Omeprazole hydroxylation is inhibited by a single dose of moclobemide in homozygotic em genotype for CYP2C19. *Br J Clin Pharmacol*. 2002;53(4):393-397.
 
-**Desta 2002** Desta, Z,., Zhao, X,., Shin, J,. G., & Flockhart DA., D. A. (2002). Clinical Significancesignificance of the Cytochromecytochrome P450 2C19 Genetic Polymorphism. *Clin*genetic polymorphism. Clinical pharmacokinetics,  *Pharmacokinet*. 2002;41(12):), 913-958.
+**Desta 2002** Desta Z, Zhao X, Shin JG, Flockhart DA. Clinical significance of the cytochrome P450 2C19 genetic polymorphism. Clin Pharmacokinet. 2002;41(12):913-58. doi: 10.2165/00003088-200241120-00002. PMID: 12222994.
```

Comment:
_Significancesignificance of the Cytochromecytochrome_ 
don't drink and write reports, lol